Makefile.am

```diff
@@ -207,14 +216,30 @@ gresources += data/moltres/moltres.gresource
 pkgdata_DATA += $(gresources)
 
 # # # MAIN LIBRARY # # #
+BUILT_SOURCES = \
+	lib/eosknowledgeprivate/ekn-resources.h \
+	lib/eosknowledgeprivate/ekn-resources.c \
+	$(NULL)
+
+ekn_resources_deps = $(shell $(GLIB_COMPILE_RESOURCES) --generate-dependencies --sourcedir=lib/eosknowledgeprivate/ lib/eosknowledgeprivate/ekn-resources.gresource.xml)
+lib/eosknowledgeprivate/ekn-resources.h: $(ekn_resources_deps)
+lib/eosknowledgeprivate/ekn-resources.c: $(ekn_resources_deps)
+
+CLEANFILES += $(BUILT_SOURCES)
+
+EXTRA_DIST += lib/eosknowledgeprivate/ekn-resources.gresource.xml
 
 eosknowledgeprivate_headers = \
 	lib/eosknowledgeprivate/ekn-util.h \
+	lib/eosknowledgeprivate/ekn-resources.h \
```

Comment:
I am not sure this needs to go in here (as opposed to the sources list below). I don't think we'll be directly calling into functions of this file from introspection.